examples/datadog-sink/README.md

```diff
@@ -0,0 +1,56 @@
+# Datadog sink example
+
+The solution helps you set up a log-streaming pipeline from Stackdriver Logging to Datadog.
+
+## Instructions
+
+1. Fill the required variables in the `terraform.tfvars` file located in this directory.
```

Comment:
I'd recommend providing a `terraform.tfvars.sample` which people can copy and edit (so a git diff isn't introduced).

Reply:
Understood -- I'll update this once I finish testing other requested changes